subjects/printstr/README.md

```diff
@@ -30,8 +30,7 @@ And its output :
 
 ```console
 $ go run . | cat -e
-Hello World!
-$
+Hello World!$ 
```

Comment:
Cannot represent the prompt with $ in this case, because it is confusing with `cat -e`.

Reply:
We will soon implement a different character to represent the prompt.

Reply:
I think the current version it's even more confusing, because `cat -e` is always supposed to print a dollar symbol at the end of line.
I'll wait for the choice of a new prompt, and then I will prepare a pull request for the affected subjects (foreach has a similar issue)